Documentation/building/windows-instructions.md

```diff
@@ -11,13 +11,20 @@ You must install several components to build the CoreCLR and CoreFX repos. These
 ## Visual Studio
 
 Visual Studio must be installed. Supported versions:
-- [Visual Studio 2015](https://www.visualstudio.com/downloads/visual-studio-2015-downloads-vs) (Community, Professional, Enterprise).  The community version is completely free.  
+- [Visual Studio 2015](https://www.visualstudio.com/vs/older-downloads/) (Community, Professional, Enterprise).  The community version is completely free.  
+- [Visual Studio 2017](https://www.visualstudio.com/downloads/) (Community, Professional, Enterprise).  The community version is completely free.  
 
-To debug managed code, ensure you have installed at least [Visual Studio 2015 Update 3](https://www.visualstudio.com/en-us/news/releasenotes/vs2015-update3-vs).
+For Visual Studio 2015:
+* To debug managed code, ensure you have installed at least [Visual Studio 2015 Update 3](https://www.visualstudio.com/en-us/news/releasenotes/vs2015-update3-vs).
+* Make sure that you install "VC++ Tools". By default, they will not be installed.
+* To build for Arm32, you need to have [Windows SDK for Windows 10](https://developer.microsoft.com/en-us/windows/downloads) installed. 
 
-Make sure that you install "VC++ Tools". By default, they will not be installed.
-
-To build for Arm32, you need to have [Windows SDK for Windows 10](https://developer.microsoft.com/en-us/windows/downloads) installed. 
+For Visual Studio 2017:
+* Make sure that you install the ".NET Core Cross-Platform Development" workload.
+* Make sure that you install the "Desktop Development with C++" workload.
+ * Make sure that the "VC++ 2017 v141 toolset (x86,x64)" component is selected in list of Optional components
+ * To build for Arm32, Make sure that the "Windows 10 SDK (10.0.14393.0)" component is selected in list of Optional components for the "Desktop Development with C++" workload.
```

Comment:
A specific version isn't required, but it is:
* Selected by default in the `Desktop Development with C++` workload
* Is easily configured using the VS2017 installer
* Has the most bug fixes and functionality of any of the SDK versions
* Is currently the default version exposed on the downloads page linked in the VS2015 section (you have to go through several more links to download an older version)